stl/inc/format

```diff
@@ -191,6 +213,32 @@ constexpr const _CharT* _Parse_width(const _CharT* _Begin, const _CharT* _End, _
     return _Begin;
 }
 
+template <class _CharT, _Parse_spec_callbacks<_CharT> _Callbacks_type>
+constexpr const _CharT* _Parse_precision(const _CharT* _Begin, const _CharT* _End, _Callbacks_type&& _Callbacks) {

```

Comment:
Pre-existing, no change requested: Consider using `_NODISCARD` on new internal machinery.

Reply:
added todo to project